parts/windows/windowscsehelper.ps1

```diff
@@ -246,6 +246,7 @@ function Get-WindowsVersion {
     switch ($buildNumber) {
         "17763" { return "1809" }
         "20348" { return "ltsc2022" }
+        "25398" { return "ltsc2022" }

```

Comment:
Please add a comment. I think that we need to remove it later since it does not make sense to return ltsc2022 for 23h2.

Reply:
Added it.